docs/config.md

```diff
@@ -42,6 +42,16 @@ Default:
 ]
 ```
 
+- `Announce`
+Array of swarm addresses to announce to the network.
+
+Default: `[]`
```

Comment:
This is a bit misleading. We announce all public addresses by default.

Reply:
Would it be better to describe this as "Array of swarm addresses to announce to the network in addition to the ones that are inferred by default."?

Reply:
If this is non-empty, we'll only announce what's in there. If it's empty, we'll announce the addresses that were inferred.

Reply:
My bad. I was mixing up the "Swarm listening" lines and the "Swarm announcing" lines of the daemon output into a single bucket.